### DISCUSSION TITLE
unable to get all the records in the table through api ?

### DISCUSSION THREAD

**Discussion by gokulgseven** [pocketbase/pocketbase#4660] [Q&A] [answered]
i am using the standard apis to get the data 
GET https://{url}/api/collections/data/records?page=1&perPage=99999 
this only gives me 500 records , i want to get all records (they should be around 10,000 no more that that)

**Comment by ganigeorgiev** [comment, marked as answer]
The max 500 records limit per paginated result is fixed and not customizable at the moment.

You can create your own custom route OR fetch your entries on batches (_the SDKs `getFullList()` will do it for you_), but note that returning 10k records as json at once could be a lot of data to keep in memory and transfer over the network so I would avoid it if possible, especially if you are planning to insert all of these items as DOM elements.
